xml/templates/drop.tpl

```diff
@@ -11,7 +11,7 @@
 --{/if}
 -- /*******************************************************
 -- *
--- * Clean up the existing tables{if isset($no_license) and $no_license} - this section generated from {$smarty.template}
+-- * Clean up the existing tables{if $isOutputLicense} - this section generated from {$smarty.template}
```

Comment:
I think this is still backwards from the original?

Reply:
So I think most of the time it should output this - but not when called from `drop.tpl` which sets this to FALSE - because for drop.tpl it's gonna be repetitive - that seems to be what the original achieves?

Reply:
Running GenCode.php this line seems still backwards.

In civicrm.mysql, the before is correct, and the after is not correct:

| Before | After | Good? |
| ------ | ----- | ----- |
| Does not output license. | Does not output license. | ✔ |
| Outputs that this section came from drop.tpl | Does not output that it came from drop.tpl | ❌ |

In civicrm_drop.mysql, ditto:

| Before | After | Good? |
| ------ | ----- | ----- |
| Does not output license. | Does not output license. | ✔ |
| Only outputs that this section came from drop.tpl once. (It misses the newline on the `-- *` and appends it to the row before but is just a spacing issue.) | Outputs that it came from drop.tpl twice. | ❌ |

If you change it on line 14 to `{if !$isOutputLicense}` then it works out the same as before.

Reply:
@demeritcowboy ok -  I swapped it

Reply:
Cool. Have marked it not-revert-on-not-fail.